deployments/mainnet/guardianSetVAAs.csv

```diff
@@ -1,3 +1,4 @@
+gs4,AQAAAAMNA9Sjem/0Nh2RcUcwgx6dSXhfYWJMjzSKnGwdgrwdmMrcXpNjOCBERcYlC7SSjz8+FlrUfKA6XWMREWii3kV2hWMBBJpd8QRk6k4ZYVif0w/BjRlwp6L/qtYX5WoPd3fyUnUlOvfRCg8PJJTcbpn8gOREq567vuJS3tLV3LUMv3pUu1oBBV9GA7VTubqeIk+cVce8o9oAq7EKvRngCBrs07NSvgYacPefXziOvlGQg47zzROi8iRZyalCBog7c5yQtA1ddGQABqj63jmX9lCjbka86x9gnt/yAasyNiJm8WbFx9pxP2oZWQwgto7T8BGcskgTxydWDt4Iaz1hDC16Hvpm9lW62QkACA9eSVp16lIkHFnRRcYWv6wB5XGCrY14TLzJhi7Tr7YMCYPMvGkFU5Yf/PEVoMkXNn2q2o5gviy7i4AIusY0GowBCTWrEeDuoouHoe3FzM4/H6wl91tfZA/msGc6fNdFE8ncAcVEIWzzZMyZk7Cf2mEuDNHO2cAPtmi4cqFqZOu1XScBCrK8OWF6I5bn3vokzXwi9C3DHzxC/82dFHKwLfhGik0FY5Eej7aktbDOC9UF2qU3ebCP9mCWezHyRhJu1/bymn4AC9ttP9ezO9yaw5kpFutKrLl+fiHRlknn+ijS3W4zeTfkJ0UWqWwTrHqIldqfkZSOo6CcJfRLmCxizohCtY4gyKkADT0bGci7AAhWtmELnSir3mw1y3cFxspdtxH3vpbWDu2dcs+kAqa/6L8EltvHrzV5b8do2lGgZ7lZQbNxLc6K4ecBDsgAhQMxV/0aVij8DFYmdGmobw5aZtfe3hrUznTsw9/5W2AwejnDv77tyRUHUHDaMNA5Xe+WNRMFhPcJs4heG9wAEPxIDrnucVotFRsjcitItCWB1/QAH8FpbHVCUEC/wf/FOU/kGK2ytkvT3Gku/aTMQIFjZ32+IzsWvNq7hToghDMBEY7p4RXhoMmB8Z0HcrhQ5mZZEyLadCqaEszp9SpWZb1HSr3VnFgAFr7oquZ/3zmzFb4lKNEu7DplKRDgPMTG+jgBEp0NHi5CnpaZGOwWPRanpbLGcpqkSvXcytB9JdGYkVVqebV09C2a29niqa5aa4dQMx0vzLMo3ZTDv4eR7hv+haoAZh6ZeBmB+uoAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAE/UxsVeyN/TQgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAENvcmUCAAAAAAAEE1iTtadsP3OWRWSIhb3MwGzXCjzT/2y5Ulib3oYsJe9DkhMvudSkIVcRTehGAZO986L8+B+GoJdl9HYv0RB6AIazLXoJd5JqIFEx2HMdOcvrjIKy/YL67ScR1Zrw8kmdFucm9rIRs5dWwEJEG+bYZQtptU6+cV4jQ1TOW000j7dLlY6JZuLsPb1JWKfNFefK8HxOPcjnxGn5LIzYj7gAWiB0o7+ROVPWlSYNiLwaolpO7jY+8AAKwAdnJ7NfvqLawo/uXMsP6naOr0XO0Ta52eJJA0ZK6In1yKcj/BT5MSS3xziEPLuJ6GTIYsOM3czPldLMN6TcA2qNIytI9izdRzFBL0iQ2nmPaJajMx9ktIwS0dV/2cvnCBFxqhvh02yv44Z5EPmcCeNHiZwZw4GStuc4fM12gnfBfasbelAnwLPPF44hrS53rgZxFUnPux+cep2AluheFIfzVRXQKpJ1NQSo11RxufSe22++vImPQD5Hc+lf6xXoDJqZyDSN
```

Comment:
Looks like the format of this file is to have this in hex

Reply:
Fixed in 04c43f17b4234b5d62575435c17d60a65541bebb